src/app/unite/unitepagination/unitepagination.component.ts

```diff
@@ -72,6 +73,9 @@ export class UnitepaginationComponent implements OnInit {
 
   checkingFuntion()
```

Comment:
Awesome. Thanks @vilbergs 
While you're at it.. do you mind fixing this typo as well. Method needs a better name in fact.